src/StakeRegistry.sol

```diff
@@ -359,11 +357,19 @@ contract StakeRegistry is StakeRegistryStorage {
                     nextUpdateBlockNumber: 0,
                     stake: newStake
                 }));
-
-                _updateQuorumRoot(operatorId, quorumNumber, uint32(block.number), newStake);
             }
         }
 
+        _updateQuorumRoot(
```

Comment:
you also need to update where the operator deregisters?

Reply:
This is the method that gets called when they deregister

Reply:
That's why arch was saying this only gets hashed with nextUpdateBlockNumber == 0

Reply:
responded above. sg.

Reply:
_recordOperatorStakeUpdate() is also called on dereregistration